Resources/templates/assets/css/modaal.css

```diff
@@ -0,0 +1,661 @@
+/*!
```

Comment:
C'est à dire ?

2016-06-03 14:48 GMT+02:00 om64 <notifications@github.com>:

> In Resources/templates/assets/css/modaal.css
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65701151>
> :
>
> > @@ -0,0 +1,661 @@
> > +/*!
>
> Il ne faut pas en tenir compte! Le 03/06/16 14:46, Godfraind a écrit :
> … <#m_9033775370283028204_>
> In Resources/templates/assets/css/modaal.css <#14 (comment)
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65700972>>:
> > @@ -0,0 +1,661 @@ > +/*! C'est normal le fichier modaal.css ? — You are
> receiving this because you authored the thread. Reply to this email
> directly, view it on GitHub <
> https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65700972>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ANpV2hls_b7kTZLh_rxzVAXYYhycEdyYks5qICI5gaJpZM4ItfhJ>.
>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65701151>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABVSttG0RqT1YCivxw6as2iTMi9SQYi-ks5qICKIgaJpZM4ItfhJ>
> .
>



-- 
Godfraind Nicolas
5, rue de la Carrière
6592 Monceau-Imbrechies
GSM: 0472 94 80 99